src/50_Samples_%26_Templates/Poll.html

```diff
@@ -0,0 +1,138 @@
+<!---{
+  "preview": "default"
+  }--->
+  <!--
+  #### Introduction
+
+  This is a sample template for a poll in AMP. After selecting one of the answers, you will see the percentage of chosen answers.
```

Comment:
Can you provide a bit more overview and context? What are the challenges you're sovling.